web/js/components/layer/product-picker/browse/browse-layers.js

```diff
@@ -263,9 +278,11 @@ function mapStateToProps(state) {
   return {
     browser,
     mode,
+    category: category ? category.title : '',
```

Comment:
Instead of having to define an additional variable inside the component, you can just name the returned prop here to `selectedCategoryName`:

`selectedCategoryName: category && category.title`

... and then access it on props with that name, e.g. `const { selectedCategoryName, ..., ..., } = props;` at the top of the component.

Reply:
Resolved in: https://github.com/nasa-gibs/worldview/pull/3783/commits/50b1ef41dfb4c212ed2a1b37313a18b937f25ad6